src/model/specific_path/Path.java

```diff
@@ -1,13 +1,67 @@
 package model.specific_path;
 
+import java.awt.Point;
+import java.util.ArrayList;
+
 import model.Shape;
 import model.ShapeName;
 
 public class Path extends Shape {
+	
```

Comment:
Actually it is not necessary opened. SVG example suggest that it can be anything.
You can do a circle, a rectangle opened or closed with a Path.
Path can do everything. Just that it is easier to just add a Circle with 3 parameters than doing one with a Path.